src/fragments/lib/datastore/ios/auth-model.mdx

```diff
@@ -83,30 +83,30 @@ In order to issue an operation (`DataStore.save`, `DataStore.delete`) on a model
 1. User's sign in state is known to be signed out, and the user needs to sign in, prior to making a call to `DataStore.save()`. 
 
 ```swift
-Amplify.Auth.signIn {
-    switch $0 {
-    case .success(let signInResult):
-        if signInResult.isSignedIn {
-            // Amplify.DataStore.save
-        }
-    case .failure(let error):
-        print("Sign in error \(error)")
+do {
+    let signInResult = try await Amplify.Auth.signIn(username: username, password: password)
+    if signInResult.isSignedIn {
+        print("Sign in succeeded")
     }
+    // Amplify.DataStore.save
```

Comment:
It seems the highlight is also copied with the text. Can you confirm it once?

Reply:
No, I don't see the highlight coped...I'm not sure why its showing dark red :/